src/main/java/com/jorndev/jorndevdemo/controllers/UserController.java

```diff
@@ -24,7 +26,7 @@ public UserController(UserService userService) {
     }
```

Comment:
Nice, constructor-based dependency injection is a good practice. Do you know why ?

There are also setter-based dependency injection and field-based dependency injection.

Can you tell the difference, pros and cons ?